api/src/shared/infrastructure/repositories/jobs/job-repository.js

```diff
@@ -27,9 +27,10 @@ export class JobRepository {
 
   /**
    * @param {Object} config
-   * @param {valueOf<JobPriority>} config.priority
-   * @param {valueOf<JobRetry>} config.retry
-   * @param {valueOf<JobExpireIn>} config.expireIn
+   * @param {string} name Job name
```

Comment:
Est-ce que ça ne serait pas plutôt comme ci-dessous ?

```suggestion
   * @param {string} config.name Job name
```